bundles/org.openhab.binding.modbus.sunspec/README.md

```diff
@@ -0,0 +1,297 @@
+```
+
+Note: make sure that refresh, port and id values are numerical, without quotes.
+
+### Item Configuration
+
+```
+Number Inverter_Temperature "Temperature [%.1f C]"  {channel="modbus:inverter-single-phase:bridge:se4000h:deviceInformation#heatsink-temperature"}
+
+Number Inverter_AC_Power "AC Power [%d W]" {channel="modbus:inverter-single-phase:bridge:se4000h:acGeneral#ac-power"}
+
+Number Inverter_AC1_A "AC Current Phase 1 [%0.2f A]" {channel="modbus:inverter-single-phase:bridge:se4000h:acPhaseA#ac-phase-current"}
+
+```
+
+### Sitemap Configuration
+
+```
+                        Text item=Inverter_Temperature
+                        Text item=Inverter_AC_Current
+                        Text item=Inverter_AC_Power
+                        Chart item=Inverter_Temperature period=D refresh=600000
+                        Chart item=Inverter_AC_Power period=D refresh=30000
+
+```
+
+## Vendor specific information
+
+### SolarEdge
+
+Newer models of SolarEdge inverters can be monitored over TCP, but you need to enable support in the inverter first.
+Refer to the "Modbus over TCP Configuration" chapter in this documentation: https://www.solaredge.com/sites/default/files/sunspec-implementation-technical-note.pdf
+
+Modbus connection is limited to a single client at a time, so make sure no other clients are using the port.
+
+## For Developers
+
+SunSpec is a big specification with many different type of devices.
+If you own or have access to an appliance that is not supported at the moment then your help is welcome.
+
+If you want to extend the bundle yourself, you have to do the followings:
+
+ - Define your thing type, channel types and channel groups according to openHAB development practices.
+ You can look at the meter and inverter types to get ideas how you can avoid repeating the same configuration over and over.
+ 
+ - Extend the `AbstractSunSpecHandler` and implement the handlePolledData method.
+ This method will be regularly called with the register data read from the appliance.
+ The method should parse the data and update the channels with them.
+
+ - The preferred way to parse the raw data is to write a parser for you model block type.
+ Your class should implement the `SunspecParser` class and it is preferred to extend the `AbstractBaseParser` class.
+ This base class has methods to accurately extract fields from the register array.
+
+ - The parser should only retrieve the data from the register array and return them in a block descriptor class.
+ Scaling and other higher level transformation should be done by the handler itself.
+ 
+ - To include your block type in auto discovery you have to add its id to the `SUPPORTED_THING_TYPES_UIDS` map in `SunSpecConstants`. This is enough for our discovery process to include your thing type in the results.
+
+
+If you have questions or need help don't hesitate to contact us.
```

Comment:
If you are going to include this line you should probably include contact information as well.

Reply:
Good point made it a bit more clear.